### PR TITLE
RFC Process

### PR DESCRIPTION
**Pull request by dwillist**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+If the change is made as a pull request, but is considered substantial or more clarity/discussion is warranted, the issue will be closed and the author will be asked to be opened.
+
+### What's in Scope
+You'll need to follow this process for anything considered "substantial".
+What constitutes a "substantial" change may include the following but is not limited to:
+- Changes to [Paketo Spec](todo.com)
+- Adding/Removing a repository to Paketo
+- Changes to the contents of the Data Format file listed [here](https://github.com/buildpacks/spec/blob/master/buildpack.md#data-format)
+- Changes that effect contents of the output image
+- process changes
```
Comment (sclevine) [inline README.md]:
```suggestion
- Process changes
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+- governance changes
```
Comment (sclevine) [inline README.md]:
```suggestion
- Governance changes
```

**Comment by dwillist** [comment]
@sclevine Believe the last commit clears up your concerns.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+	- IF the RFC effects all teams, voting members are Steering Committee Members + All Maintainers
+
+	The effect of an RFC referenced above can be defined as follows:
+
+	- Changes inside a repo a team owns effect that team
+	- If changes are proposed against a tool or utility, teams that use that utility are effected.
+	- Changes to any specification that apply to a repo effect all teams that use or maintain that repo. 
+
+- The community will discuss as much as possible in the RFC pull request directly. All discussion should be posted on the PR thread.
+- When an RFC is deemed "ready"
```
Comment (ryanmoran) [inline README.md:58]: I think this was not meant to be a bullet point.

```suggestion
When an RFC is deemed "ready"
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+- A Voting member may propose a "motion for final comment period (FCP)" along with a disposition of the outcome (merge, close, or postpone). Before entering FCP, super majority of the voting members must sign off.
+- This step is taken when enough discussion of the trade-offs have taken place and the team(s) is in a position to make a decision.
+- The FCP will last 7 days. If there's unanimous agreement among the team(s), then the FCP can close early.
+- Acceptance requires a super majority of binding votes by voting members in favor. The voting options are the following: Affirmative, Negative, and Abstinence. Non-binding votes are of course welcome. Super majority means 2/3 or greater.
```
Comment (ryanmoran) [inline README.md:62]:
```suggestion
- Acceptance requires a super majority of binding votes by voting members in favor. The voting options are the following: Affirmative, Negative, and Abstention. Non-binding votes are of course welcome. Super majority means 2/3 or greater.
```

Comment (ryanmoran) [reply README.md:62]: Additionally, I don't think I've seen any definition in here of "binding" vs. "non-binding" votes.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+- Changes that effect contents of the output image
+- Process changes
+- Governance changes
+
+For clarification about where a change fits into this model, please review previous RFCs, or reach
+out on the official Paketo slack [here](packetobuildpacks.slack.com).
+
+### Process
+#### RFCs
+To get an RFC implemented, first the RFC needs to be merged into the `rfcs`[/rfcs] repo. Once an RFC is merged, it's considered 'accepted' and may be implemented in the project. These steps will get an RFC to be considered:
```
Comment (arjun024) [inline README.md:35]:
```suggestion
To get an RFC implemented, first the RFC needs to be merged into the [`rfcs`](//github.com/paketo-buildpacks/rfcs) repo. Once an RFC is merged, it's considered 'accepted' and may be implemented in the project. These steps will get an RFC to be considered:
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+- What clearly defines a "substantial" change?
+- How long should the Final Comment Period be?
+
+## Links
+
+Slack : https://packetobuildpacks.slack.com
+
+Buildpack Spec : https://github.com/buildpacks/spec
+
+Paketo Spec : todo.com
```
Comment (arjun024) [inline README.md:94]: Doesn't look like we need this anymore

Comment (arjun024) [inline README.md:94]:
```suggestion

```
Doesn't look like we need a Paketo spec anymore

File: README.md (hunk 4 above)
Comment (dwillist) [reply README.md]: Non-binding just means you can vote, but don't expect it to count. Only voting members are given votes.

**Pull request merged**